src/mime.cr

```diff
@@ -93,15 +96,40 @@ module MIME
     ".xml"  => "text/xml; charset=utf-8",
   }
 
-  private def self.initialize_types
-    return if @@initialized
+  # Initializes the MIME database.
+  #
+  # The default behaviour is to load the internal defaults as well as the OS-provided
+  # MIME database. This can be disabled with *load_defaults* set to `false`.
+  #
+  # This method usually doesn't need to be called explicitly when the default behaviour is expected.
+  # It will be called implicitly with `load_defaults: true` when a query method
+  # is called and the MIME database has not been initialized before.
+  def self.init(load_defaults : Bool = true) : Nil
     @@initialized = true
 
-    DEFAULT_TYPES.each do |ext, type|
-      register ext, type
+    if load_defaults
+      DEFAULT_TYPES.each do |ext, type|
+        register ext, type
+      end
+
+      Crystal::System::MIME.load
```

Comment:
So how do I skip this?

Reply:
https://github.com/crystal-lang/crystal/blob/253290902ee92ae81c96f5c5ccf9a2fd856fbcfc/spec/std/mime_spec.cr#L158-L165

Reply:
No, that skips loading from `DEFAULT_TYPES`, but not reading the system db, see a couple of lines above

Reply:
`init(false)` skips both, `DEFAULT_TYPES` and `Crystal::System::MIME.load´. They're both protected by `if load_defaults`.

Reply:
Ugh yes sorry I wasn't looking right :(